kornia/feature/steerers.py

```diff
@@ -0,0 +1,66 @@
+from typing import Dict
+
+import torch
+
+from kornia.core import Module, Tensor
+from kornia.utils.helpers import map_location_to_cpu
+
+dedode_steerer_urls: Dict[str, str] = {
+    "B-C4": "https://github.com/georg-bn/rotation-steerers/releases/download/release-2/B_C4_Perm_steerer_setting_C.pth",
+    "B-SO2": "https://github.com/georg-bn/rotation-steerers/releases/download/release-2/B_SO2_Spread_steerer_setting_C.pth",
+    "G-C4": "https://github.com/georg-bn/rotation-steerers/releases/download/release-2/G_C4_Perm_steerer_setting_C.pth",
+    "G-SO2": "https://github.com/georg-bn/rotation-steerers/releases/download/release-2/G_SO2_Spread_steerer_setting_C.pth",
+}
+
+
+class DiscreteSteerer(Module):
+    def __init__(self, generator: Tensor) -> None:
```

Comment:
docstring (with example if possible) to how to init/use this class

Reply:
Thanks, fixed

Reply:
I added a more complete example to matcher here:

https://github.com/georg-bn/kornia/blob/b9975ffb74c208865fbc139ad5f619031a34b63e/kornia/feature/matching.py#L351